### ISSUE TITLE
Kibana Adapter

### ISSUE THREAD

**Issue by steckenpferd** [ioBroker/AdapterRequests#67]
Kibana is an open source data visualization plugin for Elasticsearch. It provides visualization capabilities on top of the content indexed on an Elasticsearch cluster. Users can create bar, line and scatter plots, or pie charts and maps on top of large volumes of data

https://www.elastic.co/de/products/kibana

![grafik](https://user-images.githubusercontent.com/39061622/39716598-1735024e-5231-11e8-8e47-652d53433ea2.png)

**Comment by Apollon77** [comment]
How does the data come into an elastic search?
Additionally: is Elastic search really the best technology to store time series data?
ioBroker already have an InfluxDB-Adapter and this can be used well with Grafana. WHat are the benefits of Kabana?

**Comment by steckenpferd** [comment]
https://www.elastic.co/guide/en/kibana/3.0/import-some-data.html

**Comment by Apollon77** [comment]
If elastic search then we first/together with this we need a history adapter that writes data into elastic search as databasis. Manual import is not really a solution, or ?! ;-)

**Comment by steckenpferd** [comment]
Yes, that's right - not sure if I'm the only one who knows Kibana from work therefore I requested it.

**Comment by Apollon77** [comment]
We use InfluxDB and grafana in Company and me at home too ;-)
Elastic search is kind of tricky because you need kind of a cluster and such. Heavy stuff for at home ;-)

**Comment by klein0r** [comment]
This topic is also on my list :) Worked a lot with elastic in online shops - quite easy to store data and make it visible within kibana.

**Comment by frankhirsch** [comment]
I fully agree with both of you - and would still love to see an Elasticsearch history adapter :)
Did anyone yet manage to get data into Elasticsearch (maybe using History + Filebeat)?
